### PR TITLE
OSDOCS-3705: Adding release notes for secondary scheduler operator 1.1.0

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3705

Link to docs preview:
http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3705/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html

Additional information:

### CODE REVIEW

File: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc [hunk 1]
```diff
@@ -12,6 +12,40 @@ These release notes track the development of the {secondary-scheduler-operator-f
...
+// * The maximum {product-title} version for {secondary-scheduler-operator-full} 1.1.0 is 4.11.
+
+// Need to mention anything about the updates for PSA?
+
+[id="secondary-scheduler-1.1.0-bug-fixes"]
+=== Bug fixes
+
+* TODO
+
+// This BZ is attached to the errata: https://bugzilla.redhat.com/show_bug.cgi?id=2105001
```
Comment (bergerhoffer) [inline nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:42]: @ingvagabund Do we want to list [this BZ ](https://bugzilla.redhat.com/show_bug.cgi?id=2105001) as a fixed bug for 1.1? If so, can you fill out the doc type/doc text fields please?

File: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc [hunk 2]
```diff
@@ -12,6 +12,40 @@ These release notes track the development of the {secondary-scheduler-operator-f
...
+
+// TODO: Update advisory link
+// https://errata.devel.redhat.com/advisory/97046
+
+[id="secondary-scheduler-operator-1.1.0-new-features-and-enhancements"]
+=== New features and enhancements
+
+* TODO
+// Need anything related to this?
+// * The maximum {product-title} version for {secondary-scheduler-operator-full} 1.1.0 is 4.11.
```
Comment (bergerhoffer) [inline nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:33]: @ingvagabund We pointed out the max version for v1.0.1. Is there anything new/different that we need to point out related to this for v1.1?

File: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc [hunk 3]
```diff
@@ -12,6 +12,40 @@ These release notes track the development of the {secondary-scheduler-operator-f
...
+
+// Need to mention anything about the updates for PSA?
```
Comment (bergerhoffer) [inline nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:35]: @ingvagabund What do we need to say about v1.1 and pod security admission?

Is there anything new/changed in v1.1 that we need to point out?

**Comment by bergerhoffer** [comment]
The enterprise-4.12 label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.11. And any PR going into `main` must also target the latest version branch (enterprise-4.12).

If the update in your PR does NOT apply to version 4.12 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

File: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc [hunk 4]
```diff
@@ -12,6 +12,33 @@ These release notes track the development of the {secondary-scheduler-operator-f
...
 For more information, see xref:../../../nodes/scheduling/secondary_scheduler/index.adoc#nodes-secondary-scheduler-about_nodes-secondary-scheduler-about[About the {secondary-scheduler-operator}].
 
+[id="secondary-scheduler-operator-release-notes-1.1.0"]
+== Release notes for {secondary-scheduler-operator-full} 1.1.0
+
+Issued: 2022-9-1
+
+The following advisory is available for the {secondary-scheduler-operator-full} 1.1.0:
+
+* link:https://access.redhat.com/errata/RHSA-2022:6152[RHSA-2022:6152]
```
Comment (sayjadha) [inline nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:22]: I get a 404 error when I try to access this URL. Could you please check and rectify it?

**Review by sayjadha — COMMENTED** [review]
I had just one comment about the errata link, rest LGTM.

File: nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc (hunk 4 above)
Comment (bergerhoffer) [reply nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:22]: @sayjadha Apologies, I should have stated this when I requested peer review - this URL won't work until the day the advisory ships. So I will check that it works before actually merging this PR on 9/1. Thanks!

Comment (sayjadha) [reply nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.adoc:22]: That makes sense. Thank you for the clarification!

**Comment by bergerhoffer** [comment]
@kasturinarra Can you please QE review?

http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3705/nodes/scheduling/secondary_scheduler/nodes-secondary-scheduler-release-notes.html

**Pull request merged**